Description: Correct pod NAME issue in LedgerSMB/Mailer.pm
 Is missing the dash from the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472609&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Mailer.pm
+++ b/LedgerSMB/Mailer.pm
@@ -1,6 +1,6 @@
 =head1 NAME
 
-LedgerSMB::Mailer   Mail output for LedgerSMB
+LedgerSMB::Mailer -  Mail output for LedgerSMB
 
 =head1 SYNOPSIS
 
